modules/bibconvert/lib/bibconvert_xslt_engine.py

```diff
@@ -66,6 +67,10 @@
 try:
     # lxml
     from lxml import etree
+    lxml_version = etree.LXML_VERSION
+    if lxml_version < (2, 0, 11, 0):
+        raise ImportError("Minimum lxml version supported is 2.0.11.0, available is %s" % \
+                          ('.'.join([str(v) for v in lxml_version]),))
```

Comment:
@tiborsimko @kaplun No progress has been made on my side. Let's indeed close https://github.com/inveniosoftware/invenio/pull/2275. As far as the version check goes, It seems more logical to leave the version check for `configure-tests.py`.